### PR TITLE
UIF-295 Acquisition units no longer restrict edit create or delete actions from action menu

### PR DESCRIPTION
**Pull request by NikitaSedyx**
## Purpose

https://issues.folio.org/browse/UIF-295

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/Groups/GroupDetails/GroupDetails.js [hunk 1]
```diff
@@ -51,6 +52,8 @@ const GroupDetails = ({
   const accordionStatusRef = useRef();
   const history = useHistory();
 
+  const { perms, isLoading: isPermsLoading } = useAcqRestrictions(group.id, group.acqUnitIds);
```
Comment (aliaksei-chumakou) [inline src/Groups/GroupDetails/GroupDetails.js:55]: that word `perms` might be confused with actual permissions. IMHO, we should clearly differentiate these

**Comment by aliaksei-chumakou** [comment]
@NikitaSedyx it says clearly in the ticket: "Edit and delete DO NOT appear in the actions menu". Maybe it makes sense to hide these buttons if no units

**Comment by NikitaSedyx** [comment]
@aliaksei-chumakou I'd like to ask Dennis about this, as we discussed Hide produces too many issues

File: src/Groups/GroupDetails/GroupDetails.js (hunk 1 above)
Comment (NikitaSedyx) [reply src/Groups/GroupDetails/GroupDetails.js:55]: mb restrictions as I used in hook name

**Review by axelhunn — APPROVED** [review]

**Review by aliaksei-chumakou — APPROVED** [review]

**Comment by NikitaSedyx** [comment]
coverage will be added in scope of migration to jest

**Pull request merged**